projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/monitoring/DataJobMetrics.java

```diff
@@ -206,6 +206,17 @@ public void clearGauges(final String dataJobName) {
     removeTerminationStatusGauge(dataJobName);
   }
 
+  /**
+   * Removes the delay notification and termination status gauges associated with the specific data
+   * job. This is done when a data job is being disabled.
+   *
+   * @param dataJobName The name of the data job for which to clear the gauges.
+   */
+  public void clearGaugesAtJobDisable(final String dataJobName) {
```

Comment:
You should not name a method based on who is using it but based on what it does. We may want to clear the same gauges on deploy Remove as well. 

Which is a follow-up question. Don't we ?

Reply:
Agree, the method name didn't make sense. Changed, thanks